puppet/modules/quickstack/manifests/neutron/compute.pp

```diff
@@ -127,6 +129,20 @@
       vxlan_udp_port      => $ovs_vxlan_udp_port,
       veth_mtu            => $veth_mtu,
     }
+
+    # check if opendaylight needs to be configured.
+    if ('opendaylight' in $ml2_mechanism_drivers) {
+      # OVS manager
```

Comment:
I realize you have these execs because odl seems to not yet be supported in puppet-vswitch, but I would ask that you at least extract this and the other duplicate chunk out into a simple class that takes the 2 IPs you need.  That would allow us to more easily add this to quickstack::neutron::all, which configures the network services in the more-used HA-based hostgroup.

Also, I am a bit concerned about $::ipaddress here, I am not sure that will always be the one you want it to be.  Which interface should it be using?  It can't be just any one, can it?  Maybe ovs_tunnel_iface|network or private_iface|tunnel|ip?